components/timeline-chart/src/timeline-chart-directives.ts

```diff
@@ -48,6 +48,7 @@ export class DtTimelineChartOverlayTitle {}
 export class DtTimelineChartOverlayText {}
 
 /** @internal Base class for sharing common logic between marker types. */
+@Directive()
```

Comment:
Does the selectorless directive work here?

Reply:
an abstract class that extends from a directive has to have the decorator according to the new Angular 9 this is correct.